devices/AtModem/AtModem.nuspec

```diff
@@ -27,6 +27,7 @@ This nuget allows to use AT Modem for SMS, calls, network, internal storage, HTT
       <dependency id="nanoFramework.System.IO.Ports" version="1.1.86" />
       <dependency id="nanoFramework.System.Math" version="1.5.43" />
       <dependency id="nanoFramework.System.Threading" version="1.1.32" />
+      <dependency id="nanoFramework.Iot.Device.Common.GnssDevice" version="1.0.15" />
       <dependency id="UnitsNet.nanoFramework.ElectricPotential" version="5.56.0" />
       <dependency id="UnitsNet.nanoFramework.Ratio" version="5.56.0" />
```

Comment:
Angle and Speed nugets missing?

Reply:
They are coming with the GnssDevice ;-) We then need to add just the minimum nugets. If too much, it's still ok. But the min is enough.